binaryaudit/dnf.py

```diff
@@ -90,7 +91,7 @@ def download(key, source_dir, name, old_rpm_dict):
 
 
 def generate_abidiffs(key, source_dir, new_json_file, old_json_file, output_dir,
-                      conf_dir, build_id, product_id, db_conn, cleanup=True):
+                      conf_dir, build_id, product_id, db_conn, use_suppressions, cleanup=True):
     ''' Runs abipkgdiff against the grouped packages.
```

Comment:
Perhaps instead of passing the flag it should be passing an array with the file paths, that could be handled accordingly. The array would be filled accordingly to the options passed on console, but the file paths would be attached in the ways specific to mariner or poky.

For the future, we might see to allow passing multiple suppression files through `bin/binaryaudit` as this scenario is supported by libabigail. Right now it would be just an array with the length between zero and 2.

Thanks